src/frontend/src/components/public/PIAFormTabs/collectionUseAndDisclosure/index.tsx

```diff
@@ -161,6 +199,21 @@ const PIACollectionUseAndDisclosure = () => {
       isDisable: !isMPORole(),
     },
   ];
+
+  useEffect(() => {
+    if (!deepEqual(initialFormState, collectionUseAndDisclosureForm)) {
+      piaStateChangeHandler(
+        collectionUseAndDisclosureForm,
+        'collectionUseAndDisclosure',
+      );
+    }
+  }, [
+    pia.collectionUseAndDisclosure,
```

Comment:
same for `pia.collectionUseAndDisclosure`

Reply:
missed this

Reply:
There were duplicate useEffects for two tabs which I didn't notice the first time around.